src/transformers/modeling_flax_utils.py

```diff
@@ -287,6 +287,9 @@ def params(self, params: Union[Dict, FrozenDict]):
             params = unfreeze(params)
         param_keys = set(flatten_dict(params).keys())
         if len(self.required_params - param_keys) > 0:
+            import ipdb
```

Comment:
Remember to remove this!

Reply:
oh my 😱

Reply:
Ahaha no worries - it's easy to forget! Just in case you're interested: I used to use `ipdb` for debugging, but have recently migrated to PyCharm's debugger: https://www.jetbrains.com/help/pycharm/debugging-your-first-python-application.html

I find it far easier debugging with this vs `ipdb`, since you can define all of your breakpoints without changing the code, and inspect all the variables at once. There are similar debuggers available if you use VSCode, which you might find make you more efficient than using `ipdb`. Otherwise, if you find you're already fast with `ipdb`, it's a good option!